lib/action.rb

```diff
@@ -55,7 +56,9 @@ def fetch_blob_sha(ref:, path:)
   def updated_version_file(content, level)
     version = fetch_version(content)
     updated_version = version.increment!(level.to_sym)
-    content.gsub(SEMVER_VERSION, "'#{updated_version}'")
+    quote = prefer_double_quotes ? '"' : '\''
```

Comment:
```suggestion
    quote = prefer_double_quotes ? '"' : "'"
```

Reply:
This is so funny, using single quotes on purpose while the method is `prefer_double_quotes` 😂